openshift/cirras-underwriting-api-deployment.yaml

```diff
@@ -118,6 +118,15 @@ spec:
                 configMapKeyRef:
                   name: cirras-underwriting-api-config-#{ENV}#
                   key: LOG_LEVEL
+          livenessProbe:
```

Comment:
These livenessProbe settings are different than what is currently in PROD?

Reply:
I've updated them to match prod